deploy-retool

```diff
@@ -218,22 +218,26 @@ echo " -- ${GREEN}${BOLD}!! RETOOL IS BOOTING !!${NORMAL} --"
 
 WAITED=0
 
-api_running() {
-  API_DOCKER_CONTAINER="$($MAYBE_SUDO docker ps | grep 'retool-onpremise_api')"
+api_container_up() {
+  API_DOCKER_CONTAINER = "$($MAYBE_SUDO docker ps | grep 'retool-onpremise-api')"
```

Comment:
I think you need to remove the spaces, getting this error:

```/bin/bash: line 222: API_DOCKER_CONTAINER: unbound variable```

Reply:
oops formatting habits. Fixing this